schema/test/unit/computed_columns/project_revision_total_project_value_test.sql

```diff
@@ -0,0 +1,157 @@
+begin;
+
+select plan(3);
+
+/** SETUP **/
+
+truncate cif.operator restart identity cascade;
+truncate cif.project restart identity cascade;
+insert into cif.operator (legal_name) values ('test operator');
+
+select cif.create_project();
+select cif.create_project();
+select cif.create_project();
```

Comment:
Why do we call this 3 times?

Reply:
I wanted to create 3 separate projects and test different scenarios on those projects.
1- funding parameter without max funding amount and proponent cost
2- funding parameter with max funding amount, proponent cost and additional funding sources
3- funding parameter with max funding amount and proponent cost but no additional funding sources